frontend/src/views/App/App.js

```diff
@@ -6,6 +6,7 @@ import Navigation from '../../components/Navigation/Navigation';
 import { Container } from './App.styled';
 import { GlobalStyle } from '../../assets/styles/GlobalStyles';
 import { QueryClient, QueryClientProvider } from 'react-query';
+import { BrowserRouter as Router, Switch, Route } from 'react-router-dom';
```

Comment:
Samego importa bez dodania komponentu nie powinniśmy robić ;)

Reply:
Spróbuj dodać domyślną ścieżkę :)

Reply:
` <Route exact path="/">`

Reply:
używam go w innym tasku, tu narazie nie ma zastosowania

Reply:
Jak to nie ma, jeśli instalujesz i importujesz, to wykorzystaj go ;)

Reply:
to jest tylko task do dodania npm react-router-dom, funkcjonalność będzie do niego jak zrobię push navigation

Reply:
Spróbuj ;)
A co to za task?

Reply:
Dodaj chociaż ścieżkę domyślną  ;)

Reply:
więc myślę, że moge narazie te rzeczy usunąć z App.js i zostawić tylko w tym PR dodanie reac-touter-dom do package.json

Reply:
![image](https://user-images.githubusercontent.com/41823689/150122004-0bc5ce9e-8ac7-45d3-a628-37d7c0bbd612.png)

Reply:
O tym mówimy?

Reply:
Opis taska moim zdaniem trochę źle ubrany w słowa, ale tak, to ten task, więc tu powinienem dodać react-router tylko do package.json, żebym mógł pracować nad nawigacją, a reszta nad swoimi taskami

Reply:
Dobra, ale samo zainstalowanie paczki nam nic nie da.
Możliwe, że opis jest zły tego taska.

Sprawdź jak użyć Switch i Route (5min) i dodaj od razu, bo jak każdy pododaje osobno to będą konflikty.
Raczej tak powinna brzmieć ta nazwa :)

Reply:
Instalacja paczki to nie problem, ważne, żeby po instalacji pododawać to co trzeba i skonfigurować jak trzeba, aby inni mogli pracować.

Samo zainstalowanie nie jest wyczynem i nie stanowi problemu, ani nie spowoduje konfliktów. Tutaj powinniśmy ustawić Switch i Route oraz Routera ;)

bez Routera <Link> dalej nie będzie działał ;)

Reply:
A jeśli chcemy sprawić, aby po kliknięciu w link, strona zmieniła się to też musimy dodać `<Route>`

Reply:
O to w tym tasku chodzi :)